### ISSUE TITLE
Results obtained by running the code are a lot lower than the reported ones. 

### ISSUE THREAD

**Issue by ShirleyHan6** [nlpyang/PreSumm#63]
Hi,

I used the same commands in the README to run the code. However, my reproduced results are around 0.8 lower than the reported ones for TransformerAbs and BertExtAbs. 

Am I missing anything? 

Thanks.

**Comment by ShirleyHan6** [comment]
```
xent = validate(args, device_id, cp, step)
            xent_lst.append((xent, cp))
            max_step = xent_lst.index(min(xent_lst))
            if (i - max_step > 10):
                break
        xent_lst = sorted(xent_lst, key=lambda x: x[0])[:5]
```

In the validate_abs() function in train_abstractive, the setting seems to be different from what the paper has reported?  

```
if (i - max_step > 10):
       break 
```
this could leave a lot of models unevaluated?

**Comment by nlpyang** [comment]
1. For the reproduction, I have reproduced same results with the commands.

2. This is common early stop, we stop evaluation when validation loss does not decrease for 10 steps.

**Comment by nlpyang** [comment]
Please also be aware that the paper reported averaged top-3 results, while the code is evaluating top-5.

**Comment by ShirleyHan6** [comment]
ok got it, thanks!

**Comment by ShirleyHan6** [comment]
Hi @nlpyang , I found in your paper that you used GTX 1080 Ti, may I ask if you used a docker image to run the models? 

Thanks!

**Comment by nlpyang** [comment]
no, I did not use docker

**Comment by ShirleyHan6** [comment]
Hi @nlpyang , 

Thank you very much for your answers!

Sorry to bother you again, but I tested the CNN/DM abstractive model provided in README.md on GTX1080 Ti and the result is 0.18~0.25 lower than the BertExtAbs results reported in the paper in terms of ROUGE-1, 2, and L. 

May I confirm if it is one of the best BertExtAbs checkpoints (the file name is bertsumextabs_cnndm_final_model.zip ..)? 

I used the same command as in README.md. 
```
python train.py -task abs -mode test -batch_size 3000 -test_batch_size 500 -bert_data_path BERT_DATA_PATH -log_file ../logs/val_abs_bert_cnndm -model_path MODEL_PATH -sep_optim true -use_interval true -visible_gpus 1 -max_pos 512 -max_length 200 -alpha 0.95 -min_length 50 -result_path ../logs/test_abs_bert_cnndm -test_from checkpoint_path
```
Thanks a lot.

**Comment by swift-zsw** [comment]
@ShirleyHan6  Hi，When i training TransformerABS model,I got a low rouge score. I used the same data and the same model setting as the paper report。when I load checkpoint from model_path/model_step_166000.pt. I got this result:
1 ROUGE-1 Average_R: 0.32458 (95%-conf.int. 0.31799 - 0.33093)
1 ROUGE-1 Average_P: 0.23014 (95%-conf.int. 0.22450 - 0.23555)
1 ROUGE-1 Average_F: 0.26340 (95%-conf.int. 0.25762 - 0.26874)
1 ROUGE-2 Average_R: 0.07189 (95%-conf.int. 0.06708 - 0.07690)
1 ROUGE-2 Average_P: 0.05067 (95%-conf.int. 0.04666 - 0.05502)
1 ROUGE-2 Average_F: 0.05805 (95%-conf.int. 0.05395 - 0.06258)
1 ROUGE-L Average_R: 0.29149 (95%-conf.int. 0.28493 - 0.29762)
1 ROUGE-L Average_P: 0.20639 (95%-conf.int. 0.20084 - 0.21156)
1 ROUGE-L Average_F: 0.23635 (95%-conf.int. 0.23075 - 0.24150)
Can you help me?Thanks

**Comment by zyw12358** [comment]
> @ShirleyHan6 你好，我在训练 TransformerABS 模型时，我的 rouge 分数很低。我使用与纸质报告相同的数据和相同的模型设置。当我从 model_path/model_step_166000.pt 加载检查点时。我得到了这个结果： 1 ROUGE-1 Average_R: 0.32458 (95%-conf.int. 0.31799 - 0.33093) 1 ROUGE-1 Average_P: 0.23014 (95%-conf.int. 0.22450 - 0.23555) 1 ROUGE-1 Average_F: 0.26340 (95%-conf.int. 0.25762 - 0.26874) 1 ROUGE-2 Average_R: 0.07189 (95%-conf.int. 0.06708 - 0.07690) 1 ROUGE-2 Average_P: 0.05067 (95%-conf.int. 0.04666 - 0.05502) 1 ROUGE-2 Average_F: 0.05805 (95%-conf.int. 0.05395 - 0.06258) 1 ROUGE-L Average_R: 0.29149 (95%-conf.int. 0.28493 - 0.29762) 1 ROUGE-L Average_P: 0.20639 (95%-conf .int. 0.20084 - 0.21156) 1 ROUGE-L Average_F: 0.23635 (95%-conf.int. 0.23075 - 0.24150) 你能帮我吗？谢谢

Have you solved this problem?
